### PR TITLE
changelog: add 0.1.0 details

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,19 @@ The format is based on [Keep a Changelog],
...
 
-## [0.1.0] — unreleased
+## [0.1.0] — 2020-05-12
+
+First release!
+
+### Added
+
+- Supports all instructions with the exception of floating points.
+- Passes majority of the official test suite. Does not implement all validation tests yet.
```
Comment (chfast) [inline CHANGELOG.md:16]: That's a little stretch, we skip majority of the tests.

Comment (axic) [reply CHANGELOG.md:16]: The skipped ones are floating point and malformed source (not binary) tests.

Please suggest new wording, will apply. Not only for this line but the entire changelog.

Comment (axic) [reply CHANGELOG.md]: Updated.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,21 @@ and this project adheres to [Semantic Versioning].
...
 
+First release!
+
+### Added
+
+- Supports all instructions with the exception of floating-point ones.
+- Passes a large part of the official test suite:
+  - 4482 of 4490 binary parser tests,
+  - 301 of 942 validation tests (no full validation implemented yet),
+  - 6381 skipped due to containing floating-point instructions.
```
Comment (gumb0) [inline CHANGELOG.md:19]: maybe
```suggestion
  - 6381 skipped due to containing floating-point instructions or testing text format parser.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,21 @@ and this project adheres to [Semantic Versioning].
 
 ## [0.1.0] — unreleased
 
+First release!
+
+### Added
+
+- Supports all instructions with the exception of floating-point ones.
+- Passes a large part of the official test suite:
```
Comment (axic) [inline CHANGELOG.md:16]:
```suggestion
- Passes a large part of the [official test suite (spectest 1.0)](https://github.com/WebAssembly/spec/releases/tag/wg-1.0):
```

**Pull request merged**